Archive/Wiedijk100Theorems/BallotProblem.lean

```diff
@@ -134,21 +127,17 @@ theorem counted_succ_succ (p q : ℕ) :
   · intro hl
     have hlnil := counted_ne_nil_left (Nat.succ_ne_zero p) hl
     obtain ⟨hl₀, hl₁, hl₂⟩ := hl
-    obtain hlast | hlast := hl₂ l.head! (List.head!_mem_self hlnil)
+    obtain hlast | hlast := hl₂ (l.head hlnil) (List.head_mem hlnil)
```

Comment:
The changes here are already on `master`, just we can't pre-emptively update bump/v4.8.0 to contain those because of synchronization problems with Std.